source/ranges.tex

```diff
@@ -15835,11 +15835,11 @@
 Initializes \exposid{coroutine_} with
 \tcode{exchange(other.\exposid{coroutine_}, \{\})} and
 \exposid{active_} with
-\tcode{exchange(\brk{}other.active_, nullptr)}.
+\tcode{std::move(\brk{}other.active_)}.
```

Comment:
I think it is beneficial to keep the consistency of always using `exchange` when defining a move constructor.

Reply:
> I think it is beneficial to keep the consistency of always using exchange when defining a move constructor.

This is the only place in "ranges.tex" where `exchange(` is mentioned. Every other place where we define a move-constructor, we use `std::move`; e.g. https://eel.is/c++draft/range.elements.iterator#5.sentence-1 , https://eel.is/c++draft/range.elements#sentinel-2.sentence-1 , https://eel.is/c++draft/range.take.while.sentinel#2 , https://eel.is/c++draft/range.adjacent.sentinel#2 , ...

I'm not familiar with any other place in the standard that uses `exchange` when it doesn't need to. `git grep 'Initializes.*exchange'` has zero hits. `git grep '\(^\|with \).tcode.exchange'` has only three hits, all in this paragraph.